sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Helpers/CSharpSyntaxHelper.cs

```diff
@@ -367,6 +367,9 @@ public static bool NameIs(this InvocationExpressionSyntax invocation, string nam
             return invocationName.Equals(name, StringComparison.InvariantCulture);
         }
 
+        public static bool NameIs(this IdentifierNameSyntax identifierName, string name)
+            => identifierName?.Identifier.ValueText == name;

```

Comment:
Could you validate with @andrei-epure-sonarsource that using `==` is ok here. Because I see in the previous method that we use `xxx.Equals(xxx, StringComparison.InvariantCulture)` to compare `IdentifierNameSyntax` name. So I don't understand which is the best.

Reply:
`ValueText` is a `string`, just like `name`, so in this case `==` should be fine because [operator ==](https://referencesource.microsoft.com/#mscorlib/system/string.cs,705) is just a syntactic sugar for `String.Equals`